src/platform/site-wide/mega-menu/tests/megaMenu.cypress.spec.js

```diff
@@ -64,6 +64,90 @@ const testMobileMenuSections = () => {
   cy.get('.vetnav-controller-open').contains('Menu');
 };
 
+const testMobileTabFocus = () => {
+  cy.get('[data-e2e-id="about-va-1"]').should('not.be.visible');
+  cy.get('#mega-menu-desktop #vetnav').should('not.exist');
+
+  cy.get('.vetnav-controller-open').contains('Menu');
+  cy.get('.vetnav-controller-open').click();
+  cy.get('.vetnav-controller-close').contains('Close');
+
+  // tabbing through first level menu wraps around to open/close button
+  cy.get('[data-e2e-id="mobile-home-nav-link"]')
```

Comment:
We tend to use `data-testid` in the markup and then you can use `cy.findByTestId('mobile-home-nav-link')`. `data-testid` then allows us to use the `*ByTestId()` selectors in our React Testing Library tests in addition to Cypress tests.